docs/source/conf.py

```diff
@@ -58,9 +58,10 @@ def __getattr__(cls, name):
 # built documents.
 #
 # The short X.Y version.
-version = ".".join(str(s) for s in tron.__version_info__[:2])
+version = "LATEST_VERSION"
 # The full version, including alpha/beta/rc tags.
-release = tron.__version__
+release = "LATEST_RELEASE"
```

Comment:
do these get replaced by something?

Reply:
hmm good question. I was looking at the [Paasta docs](https://github.com/Yelp/paasta/blob/master/docs/source/conf.py#L53) for reference. I'll look into it a bit more to see how it gets replaced (or if it does)

Reply:
lmao guess not https://github.com/Yelp/paasta/commit/52e2b713288e7fffaffb48dc88dc08e4c9dbff74